web/forms/implementing_simple_autocompletion/match_element.dart

```diff
@@ -17,7 +17,7 @@ class MatchElement extends PolymerElement {
   MatchElement.created() : super.created();
 
   @Observe('value, inputQuery')
-  valueChanged(_, __) => Polymer.dom(root).innerHtml = highlightedValue;
```

Comment:
I was thinking about actually deprecating this `Polymer.dom` static method... It really only exists so that the dart api lines up with the js api as much as possible, but its also really annoying how its return type is now dynamic since it can return a PolymerDom or PolymerEvent object.... thoughts?